runtime/cudaq/platform/default/rest_server/helpers/RestRemoteClient.cpp

```diff
@@ -357,6 +363,21 @@ class NvcfRuntimeClient : public RemoteRestRuntimeClient {
 public:
   virtual void setConfig(
       const std::unordered_map<std::string, std::string> &configs) override {
+    {
+      // Check if user set a specific log level (e.g., disable logging)
+      if (auto logConfigEnv = std::getenv("NVQC_LOG_LEVEL")) {
```

Comment:
This seems reasonable to me, but it would be good if @bettinaheim would weigh in here. (Maybe you've already covered this with her offline?)

Reply:
> This seems reasonable to me, but it would be good if @bettinaheim would weigh in here. (Maybe you've already covered this with her offline?)

We'll cover this in a separate PR if anything else is needed. Marking as resolved for now.